bouncerscript/constants.py

```diff
@@ -32,7 +32,7 @@
     'thunderbird': r'^(/thunderbird/releases/.*?/(update/)?(?:linux-i686|linux-x86_64|mac|win32|win64)/\:lang/(?:thunderbird|Thunderbird).*\.(?:bz2|dmg|exe|mar))$',
 }
 
-BOUNCER_PATH_REGEXES_PER_PRODUCT = {
+_BOUNCER_PATH_REGEXES_PER_PRODUCT_0 = {
```

Comment:
Nit: I'm tempted to rename this to `_BOUNCER_PATH_REGEXES_PER_PRODUCT_DEFAULT` while the one containing the `msi` to `_BOUNCER_PATH_REGEXES_PER_PRODUCT_MSI`. What do you think?